src/typeDefs/types.ts

```diff
@@ -405,6 +388,13 @@ export const types = gql`
     educationGrade: EducationGrade
     email: EmailAddress!
     employmentStatus: EmploymentStatus
+    posts(
+      after: String
+      before: String
+      first: PositiveInt
+      last: PositiveInt
+      limit: PositiveInt!
```

Comment:
`first` and `last` are the amount of records that client wants to fetch in one round trip to the server corresponding to their pagination direction

the `limit` that I asked you to be passable to `parseRelayConnectionArguments` function was to do with the allowed maximum records to be fetched in one round trip from the server, the point was to give each connection resolver the **granularity** in how many records they will return in one request

this `limit` isn't to be added to the graphql schema

Reply:
Ok sir will correct it